Sophia.md

```diff
@@ -321,13 +209,7 @@ let (_, y, _) = my3dCoordinates /* now you've retrieved y */
 
 ### Record
 [Source](https://reasonml.github.io/docs/en/record.html)
```

Comment:
We can cite Reason as an inspiration for Sophia in the introduction, but I don't think we should link the Reason documentation anywhere.